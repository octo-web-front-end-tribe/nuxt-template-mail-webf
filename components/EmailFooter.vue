<template>
  <!-- Email Footer : BEGIN -->
  <table role="presentation" cellspacing="0" cellpadding="0" border="0" align="center" width="100%"
         style="max-width: 780px;">
    <tr>
      <td
        style="padding: 40px 10px; font-family: sans-serif; font-size: 12px; line-height: 140%; text-align: center; color: #888888;"
        class="x-gmail-data-detectors">
        Tribu WebFront
        <br>
        34 avenue de l'Op√©ra, 75002 Paris
        <br>
        tribu.webf@octo.com
        <br><br>
      </td>
    </tr>
  </table>
  <!-- Email Footer : END -->
</template>

<script>
    export default {
        name: 'email-footer'
    };
</script>

<style scoped>

</style>
