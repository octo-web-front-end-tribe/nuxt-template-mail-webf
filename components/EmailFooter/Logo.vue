<template>
    <img :src="imgSrc" :alt="altText" style="width: 100px">
</template>

<script>
    import Drive from '@/services/linkDrive';

    export default {
        name: 'logo',
        props: ['driveImageID', 'alt'],
        data() {
            return {
                imgSrc: Drive.linkDrive(this.driveImageID),
                altText: this.alt
            }
        }
    };
</script>

<style scoped>

</style>
