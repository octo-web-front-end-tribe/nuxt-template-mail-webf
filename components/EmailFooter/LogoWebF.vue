<template>
    <img
            :src="srcLink"
            style="width: 100px"
    >
</template>

<script>
    import Drive from '@/services/linkDrive';

    export default {
        name: 'logo-web-f',
        data() {
            return {
                srcLink: Drive.linkDrive('13BzL0ExgsfvDpbZsTQe4qq15XS1GS0_4')
            }
        }
    };
</script>

<style scoped>

</style>
