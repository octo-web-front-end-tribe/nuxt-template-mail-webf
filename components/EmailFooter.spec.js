import Vue from 'vue'
import EmailFooter from '@/components/EmailFooter'

describe('EmailFooter Component', () => {
    let $mounted

    beforeEach(() => {
        $mounted = new Vue(EmailFooter).$mount()
    })

    test('snapshot', () => {
        let $html = $mounted.$el.outerHTML
        expect($html).toMatchSnapshot()
    })
})