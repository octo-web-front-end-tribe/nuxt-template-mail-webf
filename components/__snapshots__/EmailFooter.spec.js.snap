// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmailFooter Component snapshot 1`] = `
"<table role=\\"presentation\\" bgcolor=\\"#001849\\" cellspacing=\\"0\\" cellpadding=\\"0\\" border=\\"0\\" align=\\"center\\" width=\\"100%\\"><tr><td valign=\\"top\\" align=\\"center\\"><div class=\\"email-container\\" style=\\"max-width: 780px; margin: auto;\\"><table role=\\"presentation\\" cellspacing=\\"0\\" cellpadding=\\"0\\" border=\\"0\\" width=\\"100%\\"><tr><td style=\\"padding: 40px 10px 40px 10px; text-align: center; font-family: sans-serif; font-size: 15px; line-height: 140%; color: rgb(255, 255, 255);\\"><p style=\\"margin: 0px;\\"><img src=\\"https://drive.google.com/uc?id=13BzL0ExgsfvDpbZsTQe4qq15XS1GS0_4\\" style=\\"width: 100px;\\"> <br> <b style=\\"font-size: large;\\">There is a better Web</b> <br> <br>
                                Nous accompagnons les acteurs du Web pour construire des architectures Front
                                innovantes et de qualit√©.
                            </p></td></tr></table></div></td></tr></table>"
`;
